internal/connector/pingfederate/resources/pingfederate_oauth_client_test.go

```diff
@@ -0,0 +1,26 @@
+package resources_test
+
+import (
+	"testing"
+
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/pingfederate/resources"
+	"github.com/pingidentity/pingctl/internal/testing/testutils"
+)
+
+func TestPingFederateOAuthClientExport(t *testing.T) {
+	// Get initialized apiClient and resource
+	PingFederateClientInfo := testutils.GetPingFederateClientInfo(t)
+	resource := resources.OAuthClient(PingFederateClientInfo)
+
+	// Defined the expected ImportBlocks for the resource
+	expectedImportBlocks := []connector.ImportBlock{
+		{
+			ResourceType: "pingfederate_oauth_client",
+			ResourceName: "test",
```

Comment:
Are these supposed to just be literal "test"?

Reply:
That is how it is configured in the data.json